# is the body found by the mocap system
bool found_body

#-------------------------#
# time
float64 time

#-------------------------#
# Commands/Channels

float64 cmd_1
float64 cmd_2
float64 cmd_3
float64 cmd_4
float64 cmd_5
float64 cmd_6
float64 cmd_7
float64 cmd_8
